src/github.com/matrix-org/dendrite/clientapi/auth/tokens/tokens_handlers_test.go

```diff
@@ -0,0 +1,56 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tokens
+
+import "testing"
+
+// If any of these options are missing, validation should fail
+var invalidMissings = []string{"ServerPrivateKey", "UserID"}
+
```

Comment:
Would just add a couple more test cases for an expired token and/or invalid userID.